src/program.js

```diff
@@ -121,7 +129,8 @@ export class Program {
     argv.customPrefs = argv.pref;
 
     const runCommand = this.commands[cmd];
-
+    const configFileName = argv.config;
+    const newArgv = applyConfigToArgv(argv, configFileName, this.defaultValues);
```

Comment:
Hmm, how did Flow let you do this? It should be `applyConfigToArgv({argv, configFileName, defaultValues})`.

You will need to set up a class variable for `applyConfigToArgv` on `Program` so that it can be replaced with a stub for testing.